src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ConnectedDevicesSpec.groovy

```diff
@@ -1388,46 +1362,8 @@ srcDevices=#newSrcEnabled, dstDevices=#newDstEnabled"() {
         waitForDstDevicesInputRules(flow)
     }
 
-    //TODO: This is a candidate to be removed from this spec after default meters validation is released
```

Comment:
This validation method has been removed as the required logic is implemented in the scope of https://github.com/telstra/open-kilda/pull/3072. Verified that switch validation failed when one of the required meters was absent.

As for the switch-specific rules/meters based on switch properties(`( def switchLldpMeterCount = swProps.switchLldp ? 1 : 0) ` + for arp), the appropriate check has been added to the test `"Switch is not containing extra rules after connected devices removal"`
 
 Also new check has been added `validateLldpArpMeters` that validates each required arp/lldp rule from the list  `LLDP_POST_INGRESS_COOKIE, LLDP_POST_INGRESS_ONE_SWITCH_COOKIE, ARP_POST_INGRESS_COOKIE, ARP_POST_INGRESS_ONE_SWITCH_COOKIE`